README.md

```diff
@@ -1,12 +1,222 @@
+the need to manage both ResourceQuota and AAQ quota, or
+ClusterResourceQuota and AAQ cluster quota separately.
+
+
+## Pluggable policies for custom counting
+
+To define how you want resources to be counted for your application, you should set a sidecar container 
+in the AAQ object, where you configure the AAQ Operator. More details can be found in the "Configure AAQ Operator" section.
+
+To simplify the implementation of the sidecar container, you can use the [libsidecar](https://github.com/kubevirt/application-aware-quota-api/blob/main/libsidecar/sidecar.go) library. 
+To implement the sidecar container, you need to implement the following interface:
+
+```go
+type SidecarCalculator interface {
+    PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (corev1.ResourceList, bool, error)
+}
+```
+
+The PodUsageFunc function takes as input the pod that needs to be evaluated and all the other pods in the cluster. 
+It returns the resources that should be counted for this pod, a boolean indicating if the pod belongs to an
+application policy and an error.
+
+If a pod doesn’t match any policy, the native quota evaluation applies, just like with the standard ResourceQuota.
+
+If a pod matches multiple policies, all matching policies are evaluated, and the results are summed up.
+Please see [label-sidecar.go](https://github.com/kubevirt/application-aware-quota/blob/main/example_sidecars/label-sidecar/cmd/label-sidecar.go) for an example sidecar policy.
+for an example sidecar policy.
+
+### Configure AAQ Operator
+To configure AAQ Operator, set the following fields of the aaq object:
+
+* `aaq.spec.namespaceSelector` - determines in which namespaces AAQ's scheduling gate will be added to pods at creation time. This field follows the standard Kubernetes selector format.
+
+    ***Important Note***: if a namespace selector is not being defined, AAQ would target namespaces with the `application-aware-quota/enable-gating` label as default.
+
+    ***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace that AAQ is targeting. Such pods would fail at creation time. Therefore, users shouldn't be targeting system namespaces where `.spec.nodeName` might be set.
+    
+    For more information please see [this discussion](https://github.com/kubernetes/enhancements/issues/3521#issuecomment-2016957473)
+
+
+* `aaq.spec.configuration.allowApplicationAwareClusterResourceQuota` - (default = false) - allows creation and management of ClusterAppsResourceQuota.
+
+  ***Note***: this might lead to longer scheduling time.
+
+
+* `aaq.spec.configuration.SidecarEvaluators` - Slice of standard Kubernetes container objects. Adding containers to the SidecarEvaluators includes a 
+sidecar calculator for custom counting. These containers should mount a directory under `/var/run/aaq-sockets` called `sockets-dir` and use gRPC to send 
```

Comment:
I think we should emphasize that it's an emptyDir volume, since I assume the same volume would be used by the "main" AAQ container?

Also, why does the mount path matters here? Isn't the only thing that's important is to use the same emptyDir volume, and to place the socket in a known location inside that volume?

Maybe `libsidecar` assumes this is the location? Perhaps we should emphasize that as well / make it configurable?

```suggestion
sidecar calculator for custom counting. These containers should mount an emptyDir volume under `/var/run/aaq-sockets` called `sockets-dir` and use gRPC to send 
```

Reply:
Well i guess we can make it configurable in a future PR.
I changed the directory to emtyDir volume.

THanks!